cmd/sriovdp/manager.go

```diff
@@ -32,6 +32,7 @@ import (
 const (
 	socketSuffix = "sock"
 	netClass     = 0x02 // Device class - Network controller.	 ref: https://pci-ids.ucw.cz/read/PD/02 (for Sub-Classes)
+	fecClass     = 0x12 // Device class - Processing accelerator
```

Comment:
Please rename FecClass to a more generic name as this can appy vof devices exposing more generic baseband processing accelerator. "accClass" should be enough to distinguish from "netClass"